data/transactions/logic/eval.go

```diff
@@ -539,10 +539,16 @@ func (ep *EvalParams) RecordAD(gi int, ad transactions.ApplyData) {
 	}
 	ep.TxnGroup[gi].ApplyData = ad
 	if aid := ad.ConfigAsset; aid != 0 {
-		ep.available.createdAsas = append(ep.available.createdAsas, aid)
+		if ep.available.createdAsas == nil {
+			ep.available.createdAsas = make(map[basics.AssetIndex]struct{})
+		}
+		ep.available.createdAsas[aid] = struct{}{}
```

Comment:
Is this aid not already in the map due to the changes below?

Reply:
It is, assuming we are after the next consensus release. But I left this alone instead of wrapping it in a negation of the earlier consensus flag test.

Reply:
Well, actually, _this_ one is not, because it's the _asset_. But I answered assuming you meant the one a few lines below.